practice_app/pages/api/getAllProducts.js

```diff
@@ -0,0 +1,46 @@
+const getAllProducts = (req, res) =>
+  new Promise(async (resolve) => {
+    // Imports the Google Cloud client library
+    const vision = require("@google-cloud/vision");
+
+    // Creates a client
+    const client = new vision.ProductSearchClient();
+
+    // Google cloud project id and region name
+    const projectId = "brilliant-era-276800";
+    const location = "europe-west1";
+
+    // Resource path that represents Google Cloud Platform location.
+    const formattedParent = client.locationPath(projectId, location);
+
+    // Initializes HTML text
+    var mytext = "<h1> Products </h1>";
```

Comment:
`var` is old javascript. Use `const` for immutable variables, `let` for mutable variables.